### ISSUE TITLE
Broken webcam config

### ISSUE THREAD

**Issue by flynny75** [mkuf/prind#17]
Fluidd & Mainsail can not serve the ustreamer stream in the provided configuration. Mainsail & Fluidd are reaching out to '/webcam', but the traefik path prefix is '/stream'.

Changing the path prefix to '/webcam' is not enough, as the '/webcam' path reaches ustreamer, which doesn't know what to do with it. Changing the default ustreamer service config to:

```
  ustreamer:
    <<: *ustreamer-svc
    container_name: ustreamer
    devices:
      - /dev/video0:/dev/webcam
    labels:
      - "traefik.enable=true"
      - "traefik.http.services.webcam.loadbalancer.server.port=8080"
      - "traefik.http.routers.webcam.rule=PathPrefix(`/webcam`)"
      - "traefik.http.routers.webcam.entrypoints=web"
      - "traefik.http.middlewares.webcam.stripprefix.prefixes=/webcam"
      - "traefik.http.routers.webcam.middlewares=webcam"
```

fixes the issue, and Mainsail & Fluidd can serve the first webcam. I can submit a PR for this if required.

**Comment by mkuf** [comment]
Hi There, 

thanks for mentioning this. 

Just to clarify - the default ustreamer Service is not broken.  
Moonraker as well as Fluidd are able to display the Stream that is served via `/stream`, when set up to do so via their respective Settings. 

I'm okay with exposing ustreamer as a whole via a subpath as you suggest, instead of just the `/stream` endpoint.  
From there, Users have the freedom to use ustreamers full functionality or rely on the default paths set in the various Frontends. 

-Markus

**Comment by flynny75** [comment]
Ah, I see my mistake. I had only attempted to reconfigure Mainsail's `URL Stream` setting, not its `URL Snapshot` setting. It seems Mainsail is using the snapshot URL rather than the stream URL, which I never changed from the default `/webcam?action=snapshot`. I guess that also explains my poor webcam framerate.

I am still new to the klipper environment, thank you for your efforts with this repository. This is the third installation / deployment I have tried, and is by far the easiest
